source/contracts/Augur.sol

```diff
@@ -279,6 +280,11 @@ contract Augur is Controlled, Extractable {
         return true;
     }
 
+    function logContractAddedToWhitelist(address _addition) public onlyControllerCaller returns (bool) {
+        WhitelistAddition(_addition);
+        return true;
+    }
```

Comment:
While we are doing this, we should probably have a log for things getting added to the lookup mapping.

Reply:
Sounds reasonable. Will add